cmd/template/cluster/provider/templates/openstack/open_stack_machine_template.yaml.tmpl

```diff
@@ -0,0 +1,20 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: OpenStackMachineTemplate
+metadata:
+  labels:
+    "release.giantswarm.io/version": "{{ .ReleaseVersion }}"
+    "giantswarm.io/cluster": "{{ .Name }}"
+    "cluster.x-k8s.io/cluster-name": "{{ .Name }}"
+    "giantswarm.io/organization": "{{ .Organization }}"
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  template:
+    spec:
+      flavor: changeme
+      image: changeme
+      sshKeyName: changeme
+      cloudName: changeme
+      identityRef:
+        name: changeme
```

Comment:
Very likely same variable as above cr. Please pass in .
Default likely will be `cloud-config`.